definitions/npm/prismic.io_v3.x.x/flow_>=v0.28.x/prismic.io_v3.x.x.js

```diff
@@ -0,0 +1,331 @@
+declare module 'prismic.io' {
+  declare type linkResolver = (...args: any) => any;
+
+  declare type DaysOfWeek = 'Monday'
```

Comment:
ocd protip: You can have leading `|`s in unions:

```js
declare type DaysOfWeek = 
  | 'Monday'
  | 'Tuesday'
  | 'Wednesday'
  ...
```

You don't have to change this, but figured I'd mention it incase you like it :)